motr/io_nw_xfer.c

```diff
@@ -1071,8 +1071,7 @@ static int target_ioreq_iofops_prepare(struct target_ioreq *ti,
 		if (ioo->ioo_flags & M0_OOF_SYNC)
 			rw_fop->crw_flags |= M0_IO_FLAG_SYNC;
 		io_attr = m0_io_attr(ioo);
-		if (io_attr->oa_layout_id <= m0_lid_to_unit_map_nr);
-			rw_fop->crw_lid = io_attr->oa_layout_id;
+		rw_fop->crw_lid = io_attr->oa_layout_id;
```

Comment:
It seems the earlier 'if' statement was validating the oa_layout_id field. Is that check not required now?

Reply:
No, This check is not required in this path.